modules/network-observability-kafka-option.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-kafka-option_{context}"]
+
+= Installing Kafka (optional)
+Kafka support is provided for large scale environments. You can install Kafka from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. 
```

Comment:
maybe mention that the kafka operator in Red Hat catalog is named "Red Hat AMQ Streams"?

Reply:
Agree, I adjusted to say the Kafka Operator can be installed as Red Hat AMQ Streams from the Operator Hub. Not sure about `strimzi`.

Reply:
In another comment @stleerh says we don't need to discuss strimzi in the docs.